models/gans/wgan.py

```diff
@@ -1,12 +1,181 @@
 # TODO: Implement the wgan architecture (issue #6)
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.autograd import Variable
+from torch import autograd
+import time as t
+import matplotlib.pyplot as plt
+plt.switch_backend('agg')
+import os
+from itertools import chain
+from torchvision import utils as utils
+from utils.logging import logging
 
```

Comment:
I think we should follow PEP8 for the imports! (default package -> third party -> custom) separated with a space

Reply:
import os
import time as t
from itertools import chain

import torch
import torch.nn as nn
import torch.optim as optim
from torch.autograd import Variable
from torch import autograd
from torchvision import utils as utils
import matplotlib.pyplot as plt
plt.switch_backend('agg')

from utils import logging

Reply:
There is some dependencies that are not used like matplotlib